src/lib/viewers/controls/color-picker/ColorPickerControl.scss

```diff
@@ -0,0 +1,21 @@
+@import '~box-ui-elements/es/styles/variables';
+@import '../styles';
+
+.bp-ColorPickerControl {
+    position: relative;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 48px;
+    height: 48px;
+}
+
+.bp-ColorPickerControl-button {
+    @include bp-ControlButton;
+
+    svg {
+        background-color: #0061d5;
+        border-radius: 2px;
+        box-shadow: 0 0 0 2px white;
```

Comment:
This will create a double shadow on hover. Is that desirable?

Reply:
I don't think there will be an issue - the box-shadow here is what gives the color swatch the white border. I'm guessing the other box-shadow that you're referring to comes from the mixin that is imported above this block. That box shadow is for the entire button element so the effect that it has shouldn't cause any conflicts with the box-shadow defined in this svg element.

Reply:
Ah, okay. Should we use `border: 2px solid #fff;` here if the shadow is solid?

Reply:
Yep, that sounds good.

Reply:
Actually, on second thought, we can't use `border` here. The reason is `border` takes up the space within the bounds of the element when drawing the border whereas `box-shadow` behaves more like `outline`. I initially tried to use `outline` but then had trouble applying a border radius to the `outline`, and the solution that I found was that we could use `box-shadow` with a `border-radius` to achieve the outline effect while also having the border radius effect.